src/Types.hs

```diff
@@ -784,14 +798,25 @@ data MakeProcess
             [String]
   deriving (Generic, NFData)
 
+mpCommand :: Lens' MakeProcess (NonEmpty Char)
+mpCommand f (Shell x) = fmap (\x' -> Shell x') (f x)
+mpCommand f (Process x args) = fmap (\x' -> Process x' args) (f x)
+
 data MailcapHandler = MailcapHandler
   { _mhMakeProcess :: MakeProcess
+  , _mhCopiousoutput :: Bool
```

Comment:
OK, I think we need to define new data types instead of using Bool here.  One Bool in a product type is maybe okay.  But when your constructions start looking like `FooThing "schwa" True False False` it is hard to know what the True and False are about, and easy to make mistakes.

So I propose new data types:
```haskell
data CopiousOutput = CopiousOutput | NoCopiousOutput
data KeepTempOnExit = KeepTempOnExit | NoKeepTempOnExit -- or DiscardOnExit, or whatever
```

Reply:
That is true. Thanks for the review. Much appreciated!